airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -0,0 +1,737 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Should we add any template fields?

Reply:
Sure?   Which ones do you think would be valuable to add in here?

Reply:
All fields created in ctor.

Reply:
Should be addressed by https://github.com/apache/airflow/pull/16571/commits/4b29a39e6a9800bf69f3ebeb4cadc4a1559956ff